web/src/HUD.tsx

```diff
@@ -404,10 +405,18 @@ class HUD extends Component<HudProps, HudState> {
     let resources = (view && view.resources) || []
     let sidebarItems = resources.map(res => new SidebarItem(res))
     let isSidebarClosed = !!this.state.isSidebarClosed
+    let tiltCloudUsername = (view && view.tiltCloudUsername) || null
+    let tiltCloudSchemeHost = (view && view.tiltCloudSchemeHost) || ""
+    let tiltCloudTeamID = (view && view.tiltCloudTeamID) || null
     let sidebarRoute = (t: ResourceView, props: RouteComponentProps<any>) => {
       let name = props.match.params.name
       return (
         <Sidebar isClosed={isSidebarClosed} toggleSidebar={this.toggleSidebar}>
+          <SidebarAccount
```

Comment:
do we need to do something to prevent this from showing up in snapshots?

Reply:
Oh right! This shouldn't appear on a Snapshot.